tests/test_docs/test_code_blocks.py

```diff
@@ -137,19 +137,23 @@ class TestYamlSnippets(BaseTestDocCode):
             "code_files": [
                 "packages/valory/skills/registration_abci/fsm_specification.yaml",
                 "packages/valory/skills/safe_deployment_abci/fsm_specification.yaml",
-                "packages/valory/skills/oracle_deployment_abci/fsm_specification.yaml",
-                "packages/valory/skills/price_estimation_abci/fsm_specification.yaml",
                 "packages/valory/skills/transaction_settlement_abci/fsm_specification.yaml",
                 "packages/valory/skills/reset_pause_abci/fsm_specification.yaml",
-                "packages/valory/skills/oracle_abci/fsm_specification.yaml",
             ],
+            "skip_blocks": [2, 3, 6],
         },
         "docs/demos/simple_abci.md": {
             "code_files": ["packages/valory/skills/simple_abci/fsm_specification.yaml"],
         },
         "docs/networks.md": {"skip_blocks": [0]},
     }
 
+    skipped_files = [
+        "packages/valory/skills/oracle_deployment_abci/fsm_specification.yaml",
+        "packages/valory/skills/price_estimation_abci/fsm_specification.yaml",
+        "packages/valory/skills/oracle_abci/fsm_specification.yaml",
```

Comment:
They should no longer be part of the repo. Why skip?

Reply:
:facepalm: